frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -263,6 +264,7 @@ export default function ViewFacility() {
             <ActionButtonGroup>
               <StyledButton
                 onButtonClick={() => {
+                  dispatch(resetNewBooking())
```

Comment:
Maybe consider resetting upon booking creation success instead of reseting immediately?